QuizAppNew/View/HomeView.swift

```diff
@@ -70,15 +70,15 @@ struct HomeView: View {
         }
     }
     
-    func nazmul() {
+    func apiCall() {
     
```

Comment:
Why did you set it apiCall? what is the **API calling**? is there anything inside performing API calls?.

Reply:
no .. just use a name apicall

Reply:
changed name

Reply:
what is the API calling?

Reply:
changed name